### PR TITLE
Catapult 10.0.0.4 update

### PR DESCRIPTION
**Pull request by fboucquez**
Added new block types
Updated namespace query to use 'latest'. Added active flag back in memory comparing current height with endHeight.

### CODE REVIEW

File: catapult-sdk/src/model/ModelSchemaBuilder.js [hunk 1]
```diff
@@ -58,7 +58,12 @@ class ModelSchemaBuilder {
 				receiptsHash: ModelType.binary,
 				stateHash: ModelType.binary,
 				beneficiaryAddress: ModelType.binary,
-				feeMultiplier: ModelType.uint32
+				feeMultiplier: ModelType.uint32,
+				// optional. How to create subclasses?
+				votingEligibleAccountsCount: ModelType.uint32,
```
Comment (rg911) [inline catapult-sdk/src/model/ModelSchemaBuilder.js:63]: should it be mapped to a new model schema e.g. `ImportanceBlockFooter`?

Comment (rg911) [reply catapult-sdk/src/model/ModelSchemaBuilder.js:63]: also, if split the footer, should we bring in the block types into the EntityType?

Comment (fboucquez) [reply catapult-sdk/src/model/ModelSchemaBuilder.js:63]: It seems like this manual sdk is only used for known how to parse db fields into payloads. Afaik, It's not used for binary payload serialization and deserialization. 

ideally, this sdk would have all the logic like block type, footer, etc. This logic is already in the catbuffer generated ts. 

In the future, I would like to simplify rest by not using the sdk when moving from mongo db to payload. Just using old good typed mapping without all the formatter/model maps which are hard to follow and untyped. I would only use catbuffer (the generated ts) when we need to serialize and deserialize payloads, communicate with server api, or if we need to generate hashes for proof.

Comment (fboucquez) [reply catapult-sdk/src/model/ModelSchemaBuilder.js:63]: If rest uses the open api generated code (a rest generation version of it), the target payload types would be generated so the mapping would be hard to miss from db to payload

**Review by rg911 — DISMISSED** [review]

**Review by rg911 — APPROVED** [review]
